### ISSUE TITLE
GUI size issues

### ISSUE THREAD

**Issue by Liger0** [TASVideos/BizHawk#1930]
### Summary
The dialog windows shown in the screenshots are not behaving correctly. 
On the display configuration, the right side is cutted
On the toolbox window, it is too small so it doesn't display correctly all the icons, but what's worse is that the reduce icon is upon the program icon so there is no space at all to drag the window and move it.

### Host env.
[//]: # "List the computers you've found the bug with. If there's a version that doesn't have the bug, please put that in too. Here are some examples:"

- BizHawk dev build at 28bc7725; Win10 Home 1909; AMD Rx vega 56

![screen](https://user-images.githubusercontent.com/20385116/79152091-e4e0eb00-7dcb-11ea-9c2b-0dd6f641d58e.png)
![toolbox](https://user-images.githubusercontent.com/20385116/79152093-e5798180-7dcb-11ea-8d4a-86bc6d35329e.png)

**Comment by YoshiRulz** [comment]
CNR the three columns in the Scaling and Filtering tab (that's what's really happening, it's supposed to be two). I *could* reproduce the Tool Box' window decorations being too small, but my second row of icons is fully visible. Have you set Windows' UI scale to something other than 1x? That would explain why it's only affecting you.

(For the record, you can still move the toolbox by right-clicking the window decorations, choosing `Move`, and dragging the window decorations.)

**Comment by Liger0** [comment]
Yes I have the UI scale to 125% because the fonts are too small to be visible otherwise (in windows, generally), so I can't scale down to 100%.

**Comment by YoshiRulz** [comment]
In that case, it's a known bug where WinForms doesn't play nice with UI scale. The Tool Box already has the [auto-scale code](https://github.com/TASVideos/BizHawk/blob/7cce81a880574b76dd64b747bc78686596794e37/BizHawk.Client.EmuHawk/tools/ToolBox.Designer.cs#L55-L56) and it clearly doesn't help. I guess this issue can replace the old one (#353).

**Comment by YoshiRulz** [comment]
@Liger0 zeromus would like you to try 3b86bd741 ^, which is just an older revision of the Display Configuration dialog.

**Comment by Liger0** [comment]
Yes, the display configuration now displays correctly. The toolbox, of course, is still the same.

**Comment by Deakula** [comment]
Cross post of my reply on the referenced issue:

> As far as I can tell, this issue is still present in version 2.9.1
> 
> I have a 3 monitor setup in Windows 11. Two of those monitors are set to a resolution of 1920x1080, and the main monitor is set to a resolution of 3840x2160. Regardless of the Windows scale setting for the two 1920x1080 monitors, I observe 3 different behaviors when opening BizHawk on my main 4K monitor:
> 
> 1. When BizHawk is opened on the 4K monitor and Windows scaling is set to 100%, BizHawk, tools such as TAStudio, and all Lua forms display as expected (albeit too small to be usable with a large 4K display at 100% scaling).
>    ![Scale 100](https://user-images.githubusercontent.com/21248445/265557307-9dbc227b-d6ea-4c74-ad18-4d572523d93e.png)
>    ![TAStudio 100](https://user-images.githubusercontent.com/21248445/265558340-18e0b738-9d37-436b-a814-4bf63ecd3e6b.png)
> 2. When the Windows scaling of the 4K monitor is changed to anything but 100% (eg. 200%) after BizHawk was already opened with Windows scaling set to 100%, BizHawk, tools such as TAStudio, and all Lua forms continue to display correctly (albeit improperly scaled up causing some elements to appear blurry). This is expected behavior as many apps need to be restarted to properly respond to a scale change in Windows.
>    ![Scale 200 after 100](https://user-images.githubusercontent.com/21248445/265557308-e39c020d-a148-4d31-8b01-e26452c1f011.png)
>    ![TAStudio 200 after 100](https://user-images.githubusercontent.com/21248445/265558562-4435a57f-8e1c-49ed-9f6a-a88318d91184.png)
> 3. When BizHawk is opened on the 4K monitor while the scaling is set to anything but 100% (eg. 200%), some BizHawk GUI text (eg. FPS), tools such as TAStudio, and all Lua forms are _**not**_ displayed correctly. Lua forms in particular have incorrect proportions for drawn elements, and elements such as buttons and labels get squished and have their text cut off. In tools like TAStudio, some portions of the UI are also cut off.
>    ![Scale 200](https://user-images.githubusercontent.com/21248445/265557309-0653c38d-f103-4183-8669-9d87fd2cbe0a.png)
>    ![TAStudio 200](https://user-images.githubusercontent.com/21248445/265558341-b7287c92-2810-4dda-becb-1f7f06da36af.png)
#2604

**Comment by YoshiRulz** [comment]
Closing; leave a comment here if you notice any more differences between 100% and non-100%.
